tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -566,7 +566,8 @@ EOT
 	AGENT_DEST="${AGENT_DIR}/${AGENT_BIN}"
 
 	if [ -z "${AGENT_SOURCE_BIN}" ] ; then
-		[ "$ARCH" == "ppc64le" ] && ([ "$LIBC" == "gnu" ] || die "LIBC type for ppc64le should be gnu")
+                # LIBC type for ppc64le should be gnu
+		[ "$ARCH" == "ppc64le" ] && LIBC=gnu
```

Comment:
I'm personally against just setting it without any working. What I'd suggest doing is:
* Is it "pcc64le" and is it not using "gnu" as LIBC?
  * Then warn that LIBC **must** be "gnu", and set it as "gnu"

Does this sound reasonable?

Reply:
Agree, I will make the change.

Reply:
@fidencio 
Please review the update: https://github.com/kata-containers/kata-containers/pull/1314/commits/546760ed13bc625460a125e09be459191dbbf7bc
Thanks!